build.gradle

```diff
@@ -59,10 +59,24 @@ ospackage {
 buildRpm {
     arch = 'NOARCH'
     addParentDirs = false
-    archiveName "${packageName}-${version}.rpm"
+    finalizedBy 'renameRpm'
```

Comment:
These changes fix [an open issue](https://github.com/nebula-plugins/gradle-ospackage-plugin/issues/338) of nebula.ospackage and make the package name remain same as what it currently is.

Reply:
Does `archiveFileName` also not work as done in https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/343/files ?

Reply:
It doesn't work. There was a discussion at [here](https://amzn-aws.slack.com/archives/G018DCS00GM/p1611605393026900). [This](https://github.com/opendistro-for-elasticsearch/index-management/blob/main/build-tools/pkgbuild.gradle#L50-L71) is the solution being used.